peachjam/views/legislation.py

```diff
@@ -192,18 +192,30 @@ def get_timeline_events(self):
             point_in_time_dates = [
                 point_in_time["date"] for point_in_time in points_in_time
             ]
-            event = [
-                {
-                    "date": amendment.get("date"),
-                    "event": "amendment",
-                    "amending_title": amendment.get("amending_title"),
-                    "amending_uri": amendment.get("amending_uri"),
-                    "unapplied_amendment": bool(
-                        amendment.get("date") not in point_in_time_dates
-                    ),
-                }
-                for amendment in amendments
-            ]
+            latest_expression_date = max(point_in_time_dates)
+
+            event = []
+            for amendment in amendments:
+                event_date = amendment.get("date")
+
+                # convert string dates for comparison
+                converted_event_date = datetime.strptime(event_date, "%Y-%m-%d")
+                converted_latest_expression_date = datetime.strptime(
+                    latest_expression_date, "%Y-%m-%d"
+                )
+
+                event.append(
+                    {
+                        "date": event_date,
+                        "event": "amendment",
+                        "amending_title": amendment.get("amending_title"),
+                        "amending_uri": amendment.get("amending_uri"),
+                        "unapplied_amendment": bool(
+                            amendment.get("date") not in point_in_time_dates
+                            and converted_event_date > converted_latest_expression_date
```

Comment:
Pro-tip: you don't need to parse the dates. They're in ISO8601 format `YYYY-MM-DD`. The textual comparison of the dates is the same as the date-value comparison. You already make use of this with `max(point_in_time_dates)` above. That's why ISO8601 is so cool.

So you can just compare the string values directly.

So you can actually go back to the original list comprehension and just add `and amendment.get("date") > latest_expression_date`